src/k8s-extension/azext_k8s_extension/aaz/latest/relay/hyco/__cmd_group.py

```diff
@@ -11,6 +11,9 @@
 from azure.cli.core.aaz import *
 
 
+@register_command_group(
```

Comment:
As you only use rely command in custom function, please don’t add register_command or register_command_group